decidim-core/lib/decidim/resourceable.rb

```diff
@@ -0,0 +1,88 @@
+# frozen_string_literal: true
+require "active_support/concern"
+
+module Decidim
+  # A concern with the features needed when you want a model to be able to create
+  # links from it to another resource.
+  module Resourceable
+    extend ActiveSupport::Concern
+
+    included do
+      # An association with all the links that point to this model.
+      has_many :resource_links_to, as: :to, class_name: "Decidim::ResourceLink"
+
+      # An association with all the links that are originated from this model.
+      has_many :resource_links_from, as: :from, class_name: "Decidim::ResourceLink"
+
+      # Finds all the linked resources to or from this model for a given resource
+      # name and link name.
+      #
+      # resource_name - The String name of the resource manifest exposed by a feature.
+      # link_name     - The String name of the link between this model and the target resource.
+      #
+      # Returns an ActiveRecord::Relation.
+      def linked_resources(resource_name, link_name)
+        scope = sibling_scope(resource_name)
+
+        from = scope
+               .joins(:resource_links_from)
+               .where(decidim_resource_links: { name: link_name.to_s, to_id: id, to_type: self.class.name })
```

Comment:
This `to_s` is redundant, since if you pass a Symbol it will automatically be converted to a String when performing the query